packages/website/README.md

```diff
@@ -0,0 +1,69 @@
+# Cadl Website using 11Ty Static Website Generator
+
+This repo contains the code for static web generator for Cadl project using Eleventy(11ty) static web generator. To get started with 11Ty visit https://www.11ty.dev.
+
+Support libraries that enable the demo features are in the `packages/` directory. The demo code itself is in the `website/` directory.
+
+## Instructions for editing or contributing to this website
+
+### Prerequisites
+
+#### Clone this repository
+
+Clone this repository and cd into the root directory.
+
+#### Install project dependencies & build
+
+First, install our monorepo manager:
+
+```bash
+> npm install -g @microsoft/rush
+```
+
+Next, install the project's dependencies.
+
+```bash
+> rush install
+```
+
+Next, build the project.
+
+```bash
+> rush build
+```
+
+### Running the website
```

Comment:
I would also add a step here on how to build it for production `npm run build` saying the output will be in the `_website` folder